app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -119,11 +138,18 @@ private Boolean validateFields(@Nullable String email, @Nullable String pass) {
         return fieldsValidate;
     }
 
-    private Boolean validateEmail(@NonNull String email) {
-        Pattern pattern = Patterns.EMAIL_ADDRESS;
+    public Boolean validateEmail(@NonNull String email) {
+        String mailPattern;
+        Pattern pattern;
+        mailPattern = "^[A-Za-z0-9._%+\\-]+@[A-Za-z0-9.\\-]+\\.[A-Za-z]{2,4}$";
+        pattern = Pattern.compile(mailPattern);
```

Comment:
To validate mail's format, you can avoid these lines and use domething like this: `Patterns.EMAIL_ADDRESS.matcher(email).matches()`